Components/Decks/DeckEditor.jsx

```diff
@@ -388,6 +395,12 @@ class DeckEditor extends React.Component {
                     </Typeahead>
                     <TextArea label='Cards' labelClass='col-sm-3' fieldClass='col-sm-9' rows='10' value={ this.state.cardList }
                         onChange={ this.onCardListChange.bind(this) } />
+                    { this.state.eventId && 
+                        <h4>Please be aware: When you select this event and save this deck, you are locking it from further changes and you will not be able to edit this deck anymore</h4>
```

Comment:
"Assigning this deck to an event will mean that you will be unable to modify or delete this deck for the duration of the event"

The last part here isn't currently true, but probably should be

Reply:
good idea, changed it